### PR TITLE
Scaffolding for Studio command

### PR DESCRIPTION
**Pull request by Leland-Takamine**
Uncomment the line in App.kt to enable `maestro studio`. For now, this just opens up a web app that hits a sample `/api/hierarchy` endpoint which returns the raw maestro hierarchy output.

### CODE REVIEW

File: maestro-cli/src/main/java/maestro/cli/command/StudioCommand.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        val (maestro, _) = MaestroFactory.createMaestro(parent?.host, parent?.port, parent?.deviceId)
+
+        val port = getFreePort()
+        MaestroStudio.start(port, maestro)
+
+        val studioUrl = "http://localhost:${port}"
+        val message = ("Maestro Studio".bold() + " is running at " + studioUrl.blue()).box()
+        println()
+        println(message)
+        if (Desktop.isDesktopSupported()) {
```
Comment (dmitry-zaitsev) [inline maestro-cli/src/main/java/maestro/cli/command/StudioCommand.kt:38]: And if not?

File: maestro-studio/server/src/main/java/maestro/studio/MaestroStudio.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package maestro.studio
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import maestro.Maestro
+import org.http4k.core.Method.GET
```
Comment (dmitry-zaitsev) [inline maestro-studio/server/src/main/java/maestro/studio/MaestroStudio.kt:6]: I suggest we use Ktor instead. It is more mature and team is more familiar with it

Comment (Leland-Takamine) [reply maestro-studio/server/src/main/java/maestro/studio/MaestroStudio.kt]: updated

File: maestro-cli/src/main/java/maestro/cli/command/StudioCommand.kt (hunk 1 above)
Comment (Leland-Takamine) [reply maestro-cli/src/main/java/maestro/cli/command/StudioCommand.kt:39]: Updated the message to be clearer

**Pull request merged**